datm/cime_config/stream_definition_datm.xml

```diff
@@ -778,12 +849,12 @@
 
   <!-- Generic single point site  -->
 
-  <stream_entry name="CLM_USRDAT.$CLM_USRDAT_NAME">
+  <stream_entry name="CLM_USRDAT.NEON">
     <stream_meshfile>
       <meshfile>none</meshfile>
     </stream_meshfile>
     <stream_datafiles>
-      <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT_CLMFORC/$CLM_USRDAT_NAME/CLM1PT_data/%ym.nc</file>
+      <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT_CLMFORC/NEON/CLM1PT_data/%ym.nc</file>
```

Comment:
This will also need some adjustments to not be hardcoded.

Reply:
We'd also need some logicals in the usermod_dirs to control site-specific DATM_YR_END, since we're able to run over longer records with PRISM results.

Reply:
I'm assuming this can be a logical in the shell commands, assuming the shell commands can key off of the $CLM_USRDAT_NAME?

Reply:
@wwieder do we need a complete set of user-mods for PRISM in CTSM? We wanted to try to avoid that as much as possible.

And yes the shell_commands could key off the $CLM_USRDAT_NAME. We'll want to think about this in order to prevent as much duplication as possible. I suppose the setting of DATM_YR_* in the shell_commands could check for which CLM_USRDAT_NAME is being used and that should work. That could all be done in the same user-mod directory.

Reply:
Do we want the user to decide to turn PRISM on their own? Or do we want it by default to be turned on for specific sites?

Reply:
I think users should have to opt into PRISM.  At this point I don't think we should be deciding which input data are better for particular sites and this way users are clear when they're getting PRISM precip.  

Re. shell commands, yes, I think by keying off the $CLM_USRDAT_NAME we should be able to avoid further user-mod duplication.  At least that's my hope :)